Objects/typevarobject.c

```diff
@@ -1760,7 +1760,7 @@ PyType_Slot typevartuple_slots[] = {
     {Py_tp_methods, typevartuple_methods},
     {Py_tp_getset, typevartuple_getset},
     {Py_tp_new, typevartuple},
-    {Py_tp_iter, typevartuple_iter},
+    {Py_tp_iter, unpack_iter},
```

Comment:
Hmm, I see that we're trying to be consistent with the surrounding function definitions, but now `unpack_iter` is inconsistent with the rest of the type slots.

Reply:
Naming is hard.. I could create a dummy function `typevartuple_iter` that just calls `unpack_iter`

Reply:
How about a macro?

Reply:
Yeah that would also work :+1: